test/e2e/minor_version_compatibility.go

```diff
@@ -63,7 +63,8 @@ func MinorVersionCompatibility(logger *log.Logger) error {
 	logger.Println("Creating txsim")
 	endpoints, err := testNet.RemoteGRPCEndpoints(ctx)
 	testnet.NoError("failed to get remote gRPC endpoints", err)
-	err = testNet.CreateTxClient(ctx, "txsim", testnet.TxsimVersion, 1, "100-2000", 100, testnet.DefaultResources, endpoints[0])
+	upgradeSchedule := map[int64]uint64{}
```

Comment:
[Q] would be great to add a clarifying comments on the meaning of the keys and values in this map? I guess the key is the app version, but not sure about the value?

Reply:
That's clarified in the Go comment for the function where this parameter is used:

```
	// upgradeSchedule is a map from height to version, specifying the version to upgrade to at the given height
```